pom.xml

```diff
@@ -1214,6 +1215,12 @@
           <version>${version.net.ltgt.gwt.maven}</version>
           <extensions>true</extensions>
         </plugin>
+        <plugin>
+          <groupId>org.kie</groupId>
+          <artifactId>kie-gwthelper-maven-plugin</artifactId>
+          <version>${version.org.kie.gwthelper.maven}</version>
+          <extensions>true</extensions>
```

Comment:
Can you explain the need for this? I suspect it might have been accidentally copied from the plugin definition above and it's not actually needed.

Reply:
_removed_